tests/ssl-opt.sh

```diff
@@ -13943,7 +13943,13 @@ run_test    "Handshake defragmentation with client-initiated renegotiation: len=
             -s "Prepare: waiting for more handshake fragments 512/[0-9]\\+" \
             -s "Consume: waiting for more handshake fragments 512/[0-9]\\+" \
 
-# Test Server initiated renegotiation with fragmented handshake on TLS1.2
+# Test server-initiated renegotiation with fragmented handshake on TLS1.2
+# Note: The /reneg endpoint serves as a directive for OpenSSL's s_server
+# to initiate a handshake renegotiation.
+# Note: Adjusting the renegotiation delay beyond the library's default value
+# of 16 is necessary, as it sets the maximum record depth to match it.
```

Comment:
Ah, cool, this is what I was requesting in a previous comment (I'm reviewing commit by commit). I always like when one of my comments is addressed before even being posted :)

However, I'm not happy with the wording, in particular I don't think "record depth" makes a lot of sense. It doesn't hurt to make longer phrases and says the number of records received until the renegotiation finishes (or anything else making it completely explicit what we're talking about).

Also, I don't think there's any stack involved, and it's not about complexity. See [the documentation](https://github.com/Mbed-TLS/mbedtls/blob/development/include/mbedtls/ssl.h#L4429) of `mbedtls_ssl_conf_renegotiation_enforced()` and even its name: this is about policy not resources.